### ISSUE TITLE
Tailors Bunion

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11197]
**Preferred term label:**
Tailor's Bunion

**Synonyms**
Bunionette deformity

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0263957
Lateral bony prominence along the fifth metatarsal head, with or without varus deviation of the small toe

Radiologically, foot radiographs showing varus deformity at the 5th metatarsophalangeal (MTP) joint as measured by increased 4th-5th intermetatarsal angle (IMA, >8°), increased lateral deviation angle (>7°), or increased width of 5th metatarsal head (>14mm)
https://radiopaedia.org/articles/bunionette
https://www.orthobullets.com/foot-and-ankle/7017/bunionette-deformity

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Bunionette toe deformity

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001298
- label: Bunionette deformity
- parents: [Abnormality of the fifth metatarsal bone (HP:0008089)](https://hpo.jax.org/app/browse/term/HP:0008089)
- pmid: PMID:17478753
- def: The bunionette, or tailor's bunion, is a lateral prominence of the fifth metatarsal head. Most commonly, bunionettes are the result of a widened 4-5 intermetatarsal angle with associated varus of the metatarsophalangeal joint.
- comment: Radiologically, foot radiographs showing varus deformity at the 5th metatarsophalangeal (MTP) joint as measured by increased 4th-5th intermetatarsal angle (Imore than 8 degrees), increased lateral deviation angle (more that 7 degrees), or increased width of 5th metatarsal head (more than 14mm).
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11197
- synonyms: Tailor's bunion(NONE)|Bunionette toe deformity(NONE)
